### DISCUSSION TITLE
CORS Settings

### DISCUSSION THREAD

**Discussion by virgilbugnariu** [pocketbase/pocketbase#606] [Q&A] [answered]
Hello,

Not sure if I'm missing something, and search didn't brought up anything related to CORS configuration.
Since I switched to 80 and 443 ports I started getting CORS errors. 
Is there any additional configuration that needs to be made when using these ports?
Does the `Settings > Application > Application URL` property affect CORS rules in any way?

Thanks 👍 

Later Edit:
Looks like the error was caused by the HTTPS redirect. It was caused by the fact that I was trying to use the non secure URL locally. Switched to the secure URL and it worked.

**Comment by ganigeorgiev** [comment, marked as answer]
> Does the Settings > Application > Application URL property affect CORS rules in any way?

No, this is primarily used in the mail templates as placeholder.

I'm glad that you've figured out what is the reason of the error.

Just for info, PocketBase by default allows all origins (`*`), unless you've started the executable with the `--origins=...` flag.

**Reply by virgilbugnariu** [reply]
Good to know :) Thank you
